packages/react-native-reanimated/src/logger/logger.ts

```diff
@@ -117,7 +117,7 @@ type LogOptions = {
 };
 
 function handleLog(
-  level: Exclude<LogBoxLogLevel, 'syntax'>,
+  level: Exclude<LogBoxLogLevel, 'syntax' | 'fatal'>,
```

Comment:
We don't use `fatal` for now, so it's better to remove it to prevent confusion. I added it just because logbox has a `fatal` type separate from  `error`/`warn`/`syntax` types.